.github/workflows/android.yml

```diff
@@ -78,6 +78,14 @@ jobs:
           path: /tmp/android-ndk-r16b
           key: android-ndk-${{ matrix.os }}-r16b
 
+      - name: Update homebrew (avoid bintray errors)
+        if: startsWith(matrix.os, 'macos')
```

Comment:
any particular reason this uses startsWith(matrix.os) and the below uses runner.os ==? Can you unify them?

Reply:
So I was trying to keep it consistent with any other macos checks in those blocks. 
For a couple of the blocks, I found the comparison used runner.os and for others, we used macos.

Reply:
For example, cpp-packaging uses `runner.os` comparison,
https://github.com/firebase/firebase-cpp-sdk/commit/c87c8a4abf71d3c31f8f29fd9b4ca04d68b289f2

integration_tests.yml uses "matrix.os"
https://github.com/firebase/firebase-cpp-sdk/commit/ce9c9fa52ae3f96f43971b2040b545c439efbc00